doc/user_manual/model.tex

```diff
@@ -1323,9 +1322,14 @@ \subsection{EnsembleModel}
         The user can specify as many \xmlNode{Input} as required by the sub-model.
         \nb All the inputs here specified need to be listed in the Steps where the EnsembleModel
         is used.
+     \item \xmlNode{Output}, \xmlDesc{string, optional field},
+        represents the output entities that need to be linked to this sub-model. This XML block(s) can
+        be omitted since it (they) is (are) optional.  \nb The \xmlNode{Output}s here specified are not part 
```

Comment:
this line can be omitted since it is a tautology :-P : 
This XML block(s) can be omitted since it (they) is (are) optional